src/client/components/thumbnailContainer/thumbnailContainer.css

```diff
@@ -0,0 +1,16 @@
+.thumbnail_container {
+    padding-right: 0px !important;
```

Comment:
Can we make the CSS rule more specific and avoid the !important declaration? It would be nice to not have to add those, especially early in the project.

Reply:
I'm going to land this then fix this in a followup